mintpy/ifgram_inversion.py

```diff
@@ -196,6 +198,21 @@ def cmd_line_parse(iargs=None):
 
     inps.timeseriesFile, inps.tempCohFile = inps.outfile
 
+    # --config option
+    inps.config = inps.config.lower()    # use lowercase all the time
+    if inps.config == 'no':
+        inps.config = ipns.cluster.lower()
+
+    if inps.parallel and inps.config != inps.cluster.lower():
+        import dask
+        try:
+            dask.config.get('jobqueue.{}'.format(inps.config))
+        except KeyError:
```

Comment:
If the custom input dask configuration not found. Is it better to `raise ValueError()` and exit the program, or to just printout a warning message and continue?

I usually prefer to raise exception and exit, so that the user can be more aware of the type errors and avoid using unnecessary computing time.

Reply:
Since there is a sensible fallback option, I would rather use the fallback than violently terminate the program. I figure the printed message should be sufficient.